be/src/formats/json/binary_column.cpp

```diff
@@ -122,4 +122,14 @@ Status add_native_json_column(Column* column, const TypeDescriptor& type_desc, c
     return Status::OK();
 }
 
+Status add_native_json_column(Column* column, const TypeDescriptor& type_desc, const std::string& name,
```

Comment:
This implementation is same with above `add_native_json_column`.

Reply:
This is an ugly workaround.

These two functions accept different parameter, one is `simdjson::ondemand::value`, another is `simdjson::ondemand::object`, that's the difference.

So you may think, why convert `value` to `object`, or `object` to `value`? I could not find an API of simdjson which implement it, and both `value` and `object` are sealed, not extensible.

Some other explaination was written at  `json_scanner.cpp:542`.